### ISSUE TITLE
Trying to install with the latest jupyterlab fails

### ISSUE THREAD

**Issue by daleevans** [mlshapiro/jupyterlab-flake8#8]
Is this really a hard dependency on Jupyterlab 0.17?

```
Step 27/29 : RUN jupyter labextension install jupyterlab-flake8
 ---> Running in 15d74b28742f
> /usr/bin/npm pack jupyterlab-flake8
jupyterlab-flake8-0.2.2.tgz

Errored, use --debug for full output:
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.18.3 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/application
>=0.18.3 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/apputils
>=0.18.3 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/fileeditor
>=0.18.3 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/notebook
>=0.18.3 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/terminal
The command '/bin/sh -c jupyter labextension install jupyterlab-flake8' returned a non-zero code: 1
```

**Comment by mlshapiro** [comment]
Sorry for the delay - I will fix this.

**Comment by mlshapiro** [comment]
fixed in v0.2.3
